src/wasm-binary.h

```diff
@@ -261,17 +261,32 @@ enum ASTNodes {
   F64Le = 0x9a,
   F64Gt = 0x9b,
   F64Ge = 0x9c,
```

Comment:
But in [the spec](https://docs.google.com/document/d/1-G11CnMA0My20KI9D7dBR6ZCPOBCRD0oCH6SHCPFGx0/edit), it looks like the way it was, not the way you changed it to?